All time Internal Server Error

I'll trying to make simple request:
```shell
curl -X POST \
  http://127.0.0.1:5000/pois \
  -H 'Content-Type: application/json' \
  -d '{
  "request": "pois",
  "geometry": {
    "bbox": [
      [32.70149630013472, -96.72892645140573],
      [32.70137085098112, -96.72854256023189],
      [32.710300884253556, -96.72420399392107]
    ] 
  }
}'
```
But all time i'll got status 500 and:
```html
<html>
  <head>
    <title>Internal Server Error</title>
  </head>
  <body>
    <h1><p>Internal Server Error</p></h1>
    
  </body>
</html>
```
Is it wrong bbox, or error with openpoiservice? How i can check it?

@zephylac no, nothing like it. But there are lot of errors like this:
```shell
[2020-02-20 07:16:58 +0000] [10] [ERROR] Error handling request /
Traceback (most recent call last):
  File "/ops_venv/lib/python3.6/site-packages/flask/app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "/ops_venv/lib/python3.6/site-packages/flask/app.py", line 1590, in dispatch_request
    self.raise_routing_exception(req)
  File "/ops_venv/lib/python3.6/site-packages/flask/app.py", line 1573, in raise_routing_exception
    raise request.routing_exception
  File "/ops_venv/lib/python3.6/site-packages/flask/ctx.py", line 294, in match_request
    self.url_adapter.match(return_rule=True)
  File "/ops_venv/lib/python3.6/site-packages/werkzeug/routing.py", line 1945, in match
    raise NotFound()
werkzeug.exceptions.NotFound: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/ops_venv/lib/python3.6/site-packages/flask/app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "/ops_venv/lib/python3.6/site-packages/flask/app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/ops_venv/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/ops_venv/lib/python3.6/site-packages/flask/app.py", line 1512, in handle_user_exception
    return self.handle_http_exception(e)
  File "/ops_venv/lib/python3.6/site-packages/flask/app.py", line 1471, in handle_http_exception
    return handler(e)
  File "/deploy/app/openpoiservice/server/__init__.py", line 82, in page_not_found
    return jsonify({"error_message": 404})
  File "/ops_venv/lib/python3.6/site-packages/flask/json.py", line 251, in jsonify
    if current_app.config['JSONIFY_PRETTYPRINT_REGULAR'] and not request.is_xhr:
  File "/ops_venv/lib/python3.6/site-packages/werkzeug/local.py", line 347, in __getattr__
    return getattr(self._get_current_object(), name)
AttributeError: 'Request' object has no attribute 'is_xhr'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/ops_venv/lib/python3.6/site-packages/gunicorn/workers/async.py", line 56, in handle
    self.handle_request(listener_name, req, client, addr)
  File "/ops_venv/lib/python3.6/site-packages/gunicorn/workers/ggevent.py", line 152, in handle_request
    super(GeventWorker, self).handle_request(*args)
  File "/ops_venv/lib/python3.6/site-packages/gunicorn/workers/async.py", line 107, in handle_request
    respiter = self.wsgi(environ, resp.start_response)
  File "/ops_venv/lib/python3.6/site-packages/flask/app.py", line 1997, in __call__
    return self.wsgi_app(environ, start_response)
  File "/ops_venv/lib/python3.6/site-packages/flask/app.py", line 1985, in wsgi_app
    response = self.handle_exception(e)
  File "/ops_venv/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/ops_venv/lib/python3.6/site-packages/flask/app.py", line 1547, in handle_exception
    return self.finalize_request(handler(e), from_error_handler=True)
  File "/deploy/app/openpoiservice/server/__init__.py", line 86, in server_error_page
    return jsonify({"error_message": 500})
  File "/ops_venv/lib/python3.6/site-packages/flask/json.py", line 251, in jsonify
    if current_app.config['JSONIFY_PRETTYPRINT_REGULAR'] and not request.is_xhr:
  File "/ops_venv/lib/python3.6/site-packages/werkzeug/local.py", line 347, in __getattr__
    return getattr(self._get_current_object(), name)
AttributeError: 'Request' object has no attribute 'is_xhr'

```

--- a/.travis.yml
+++ b/.travis.yml
@@ -11,7 +11,7 @@ addons:
   postgresql: "9.5"
   apt:
     packages:
-    - postgresql-9.5-postgis-2.3
+    - postgresql-9.5-postgis-2.4
 
 before_install:
   - sudo apt-get update

--- a/requirements.txt
+++ b/requirements.txt
@@ -16,3 +16,4 @@ gunicorn==19.7.1
 flask-cors==3.0.3
 gevent==1.3.5
 git+https://github.com/timmccauley/imposm-parser.git@python3
+werkzeug>=0.15,<1.0

